src/main/java/com/google/research/bleth/servlets/ReadHardCodedEstimatedBoardStateServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.research.bleth.servlets;
+
+import com.google.research.bleth.services.DatabaseService;
+import com.google.research.bleth.simulator.Board;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
```

Comment:
The imports within each block should appear in ASCII sort order. https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing